lib/core/src/tab/tab.tsx

```diff
@@ -28,14 +28,36 @@ interface TabStyle {
 }
 
 interface Props {
+	/**
+	 * Title and Body of each tab.
+	 */
 	children: Tab[];
+	/**
+	 * Define if tabs padding is removed.
+	 */
 	noPadding?: boolean;
+	/**
+	 * Declare the style of tabs. By default, style is Tabs.style.outset but
+	 * you can change it to Tabs.styles.flat.
+	 */
 	style?: TabStyle;
+	/**
+	 * Define if tabs height is inherited from parents component.
+	 */
 	fullHeight?: boolean;
 	// Uncontrolled
+	/**
+	 * Declare which tab is the initial tab.
+	 */
 	initialTab?: string;
 	// Controlled
+	/**
+	 * Declare which tab is currently active.
+	 */
 	activeTab?: string;
+	/**
+	 * Set which tab is activated.
+	 */
```

Comment:
This is not really a function that "set" the active tab. It's more like a "handler". Something like "A handler to set the activated tab". We want to emphasize that this is just a passive handler to handle the event from the Tabs component